app/src/main/java/io/github/droidkaigi/confsched2017/util/LocaleUtil.java

```diff
@@ -20,10 +25,51 @@
 
     public static final String LANG_EN = "en";
 
+    public static final List<String> SUPPORT_LANG = Arrays.asList(LANG_JA, LANG_EN);
```

Comment:
Currently, we are going to manage languages by `Locale`, not `String`.
But it's okay to change it on another PR.